src/lib/annotations/image/ImageAnnotator.js

```diff
@@ -105,7 +105,7 @@ class ImageAnnotator extends Annotator {
             type
         };
 
-        if (!annotatorUtil.validateThreadParams(threadParams)) {
+        if (annotatorUtil.validateThreadParams(threadParams)) {
```

Comment:
Ah! I was forcing this to trigger the emission of annotation errors. Will fix that up